### PR TITLE
Presence | DevTR

### PR DESCRIPTION
**Pull request by eren885**
image
image

### CODE REVIEW

File: websites/D/DevTR/presence.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  clientId: "749319733807153162"
+});
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  };
+
+  const page = window.location.pathname
+  const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/D/DevTR/presence.ts:11]: merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/D/DevTR/presence.ts (hunk 1 above)
Comment (eren885) [reply websites/D/DevTR/presence.ts]: why

Comment (Bas950) [reply websites/D/DevTR/presence.ts]: rules

**Review by doomlerd — CHANGES_REQUESTED** [review]
Update your fork.

File: websites/D/DevTR/presence.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const presence = new Presence({
+    clientId: "749319733807153162"
+});
+presence.on("UpdateData", () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "logo"
+    };
+    const page = window.location.pathname,
```
Comment (Bas950) [inline websites/D/DevTR/presence.ts:8]: combine consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
@eren885 
Combining consts:
```
 const presenceData: PresenceData = {
        largeImageKey: "logo"
    },
     page = window.location.pathname,
    browsingStamp = Math.floor(Date.now() / 1000);
```

**Pull request closed without merging**